### ISSUE TITLE
[UI] Wireframe Design for homepage

### ISSUE THREAD

**Issue by Kajol-Kumari** [anitab-org/vms#1034]
## Description
Currently we have a simple home page with a logo, few options and a basic introduction.
## Mocks
![imageedit_2_7230070166](https://user-images.githubusercontent.com/44888949/91129368-8c015780-e6c7-11ea-8ddf-33c08a2ce892.png)

### Update
- [ ] Create a proper Home page UI using wireframe containing proper header, footer and other functionalities in a well organised and more attractive way.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2 hours

**Comment by pallavithakare28** [comment]
I would like to work on this. Will me please guide me to proceed with this? As I am new to open source

**Comment by Kajol-Kumari** [comment]
Hey @pallavithakare28! Please accept my apologies but according to the new guidelines we received, we will count `First come first serve` on the basis of comments/claims done 1st september onwards. So, everyone who are interested in this issue, do come again on 1st and comment first to get assigned.

**Comment by pallavithakare28** [comment]
Ok Sure, Thank you

**Comment by pallavithakare28** [comment]
@Kajol-Kumari I would like to work on this. Will you please guide me to proceed with this? As I am new to open source

**Comment by Kajol-Kumari** [comment]
@pallavithakare28 please go ahead with this issue :+1:

**Comment by Anmollenka** [comment]
@pallavithakare28 Can I work on this issue?

**Comment by Kajol-Kumari** [comment]
Hey @Anmollenka, @pallavithakare28 is already assigned this issue.so please feel free to claim any other issue.

**Comment by Anmollenka** [comment]
@Kajol-Kumari  Can you assign it to me when it's free??

**Comment by pallavithakare28** [comment]
> @pallavithakare28 Can I work on this issue?
@Anmollenka Yes, I am working on the issue and hoping to get a resolution soon.  
Thank you:)

**Comment by khushishikhu** [comment]
> @khushishikhu This issue is already assigned to @pallavithakare28. Please do check once before claiming and feel free to claim any other issue. @pallavithakare28 can you please provide the update on the work you have done so far for this issue as you haven't opened any PR and it's 3rd day already.

Actually i was claiming it because i thought that there isn't any activity going on from 3 days . So may be you could assign it to someone else who is interested to work on it.

**Comment by pallavithakare28** [comment]
> @khushishikhu This issue is already assigned to @pallavithakare28. Please do check once before claiming and feel free to claim any other issue. @pallavithakare28 can you please provide the update on the work you have done so far for this issue as you haven't opened any PR and it's 3rd day already.

@Kajol Kumari Just now I have opened a PR mam. I am trying to work on it and looking forward to making attractive changes.

**Comment by pallavithakare28** [comment]
@Kajol-Kumari Hello Mam,  Please help to proceed further. Am I doing it in the correct way.? As I am new to Open-source but willing to contribute. Please guide me.

**Comment by Kajol-Kumari** [comment]
Hey @pallavithakare28 the issue is for just making the UI/UX design not for coding it down. So you just need to create the UI/UX design using any tool and put the final image under `wireframes` folder(create this folder as we don't have it currently) and make the PR.

**Comment by pallavithakare28** [comment]
@Kajol-Kumari Mam I have made the changes using HTML and CSS also. And created a new template. May I post it here.

**Comment by Kajol-Kumari** [comment]
> @Kajol-Kumari Mam I have made the changes using HTML and CSS also. And created a new template. May I post it here.

@pallavithakare28 please put those templates under the [#1085](https://github.com/anitab-org/vms/pull/1085) PR description.

**Comment by pallavithakare28** [comment]
@Kajol-Kumari Yes mam, I will share a screenshot here first and please suggest me the change that I have to make. Then I will post the template.
